conf/article_validator.yaml

```diff
@@ -0,0 +1,721 @@
+        properties:
+          materialLocation:
+            type: string
+          code:
+            type: string
+            minLength: 3
+            maxLength: 3
+            pattern: /[0-9]{3,3}/
+      additionalColorCodes:
+        description: |
+          These are the color codes of the colors most (ordered by most used) used on the respective
+          article simples found in this ArticleConfig instance. Optional. Must be set if the
+          ArticleSimples of this ArticleConfig utilize more than one Color.
+        type: array
+        items:
+          type: string
+          minLength: 3
+          maxLength: 3
+          pattern: /[0-9]{3,3}/
+        maxItems: 2
+      articleConfigAttributes:
+        description: |
+          Collection of Product
+          Attributes - if present at least one
+          Attribute is
+          required
+        type: array
+        items:
+          $ref: '#/definitions/SilhouetteKeyValuePairs'
+        minItems: 1
+
+  SimpleSchemaRoot:
+    description: Root object representation of a simple schema
+    allOf:
+    - $ref: '#/definitions/Root'
+    type: object
+    properties:
+      data:
+        description: The schema object
+        $ref: '#/definitions/SimpleSchema'
+
+  SimpleSchema:
+    type: object
+    description: |
+      Schema representation of an article simple: a purchasable incarnation inheriting from model and config.
+    required:
+      - ean
+      - partnerArticleSimpleId
+      - partnerSKU
+      - size
+    properties:
+      # --- Required properties ------------------------------------------------------------------->
+      ean:
+        description: |
+          European Article
+          Number - EAN required for Matching
+          with Zalando
+          Inventory
+          MUST BE
+          GLOBALLY UNIQUE
+        type: string
+        minLength: 11
+        maxLength: 13
+        pattern: /[0-9]*/
+      partnerArticleSimpleId:
+        description: |
+          External Article
+          Unique Identifier from Partner -
+          MUST BE GLOBALLY
+          UNIQUE
+          MUST BE
+          IMMUTABLE
+        type: string
+      partnerSKU:
+        description: |
+          External Article
+          Unique Identifier from Partner -
+          MUST BE GLOBALLY
+          UNIQUE
+        type: string
+      size:
+        type: string
+      # --- Optional properties ------------------------------------------------------------------->
+      length:
+        type: string
+        description: |
+          Length of
+          ArticleSimple - must match the
+          respective LengthGrid provided
+          on
+          ArticleModel
+      articleSimpleAttributes:
+        description: |
+          Collection of Product
+          Attributes - if present at least one
+          Attribute is
+          required
+        type: array
+        items:
+          $ref: '#/definitions/SilhouetteKeyValuePairs'
+
+      price:
```

Comment:
see existing api endpoints for price & stock; if at all, aggregate these endpoints in one for convenience